setup.py

```diff
@@ -44,8 +44,9 @@
 TESTS_REQUIRE = ['nose', 'pytest', 'pytest-cov', 'pytest-mock',
                  'pytest-timeout', 'pytest-rerunfailures', 'pytest-remotedata']
 EXTRAS_REQUIRE = {
-    'optional': ['ephem', 'cython', 'netcdf4', 'nrel-pysam', 'numba',
-                 'pvfactors', 'scipy', 'siphon', 'tables', 'cftime >= 1.1.1'],
+    'optional': ['cython', 'ephem', 'netcdf4', 'nrel-pysam', 'numba',
+                 'pvfactors', 'scipy', 'siphon', 'statsmodels', 'tables',
```

Comment:
should we specify a minimum version of statsmodels? probably only needed if the feature is not available in a release that occurred within the last few years.

Reply:
I don't think so, the only capability being used is the Robust Linear Model (RLM) class which I would guess is stable.